seqr/views/utils/permissions_utils.py

```diff
@@ -19,14 +23,48 @@ def get_project_and_check_permissions(project_guid, user, **kwargs):
     return project
 
 
+def project_has_anvil(project):
+    return anvil_enabled() and bool(project.workspace_namespace and project.workspace_name)
+
+
+def _map_anvil_seqr_permission(anvil_permission):
+    if anvil_permission['pending']:
+        return None
+    if anvil_permission['accessLevel'] in ['WRITER', 'OWNER', 'PROJECT_OWNER']:
+        return CAN_EDIT
+    return CAN_VIEW if anvil_permission['accessLevel'] == 'READER' else None
+
+
+def anvil_has_perm(user, permission_level, project):
+    if not project_has_anvil(project):
+        return False
+    workspace_permission = user_get_workspace_access_level(user, project.workspace_namespace, project.workspace_name)
+    if not workspace_permission:
+        return False
+    workspace_permission['pending'] = False
```

Comment:
why is this line needed? If the "pending" attribute isn't set here, you can always change the check in `_map_anvil_seqr_permission`  from `anvil_permission['pending']` to `anvil_permission.get('pending')`

Reply:
Good point.